client/my-sites/checkout/composite-checkout/hooks/use-create-payment-complete-callback.tsx

```diff
@@ -1,3 +1,4 @@
+import { SiteDetails } from '@automattic/data-stores/src';
```

Comment:
Should be fine to import this from `@automattic/data-stores`. You could also use `import type` since this only imports a type.

Reply:
I was going to say the same thing: we shouldn't import things from deep inside a package as such things are private implementation details and might change without warning.